test/powershell/Modules/Microsoft.PowerShell.Utility/Remove-Alias.Tests.ps1

```diff
@@ -0,0 +1,46 @@
+Describe "Remove-Alias" -Tags "CI" {
+    It "Remove-Alias should remove a non-readonly alias"{
+        {
+            Set-Alias -Name "foo" -Value "bar" -ErrorAction Stop
+            Remove-Alias -Name "foo" -ErrorAction Stop
+            Get-Alias -Name "foo" -ErrorAction Stop
+        } | ShouldBeErrorId 'ItemNotFoundException,Microsoft.PowerShell.Commands.GetAliasCommand'
+    }
+
+    It "Remove-Alias should throw on a readonly alias"{
+        {
+            Set-Alias -Name "foo" -Value "bar" -Option ReadOnly -ErrorAction Stop
+            Remove-Alias -Name "foo" -ErrorAction Stop
+        } | ShouldBeErrorId 'AliasNotRemovable,Microsoft.PowerShell.Commands.RemoveAliasCommand'
+    }
+
+    It "Remove-Alias should remove a non-readonly alias with force"{
+        {
+            Set-Alias -Name "foo" -Value "bar" -ErrorAction Stop
+            Remove-Alias -Name "foo" -Force -ErrorAction Stop
+            Get-Alias -Name "foo" -ErrorAction Stop
+        } | ShouldBeErrorId 'ItemNotFoundException,Microsoft.PowerShell.Commands.GetAliasCommand'
+    }
+
+    It "Remove-Alias should remove a readonly alias with force"{
+        {
+            Set-Alias -Name "foo" -Value "bar" -Option ReadOnly -ErrorAction Stop
+            Remove-Alias -Name "foo" -Force -ErrorAction Stop
+            Get-Alias -Name "foo" -ErrorAction Stop
+        } | ShouldBeErrorId 'ItemNotFoundException,Microsoft.PowerShell.Commands.GetAliasCommand'
+    }
+
+    It "Remove-Alias should throw if alias does not exist"{
+        {
+            Get-Alias -Name "foo" -ErrorAction SilentlyContinue | Should BeNullorEmpty
+            Remove-Alias -Name "foo" -ErrorAction Stop
+        } | ShouldBeErrorId 'ItemNotFoundException,Microsoft.PowerShell.Commands.RemoveAliasCommand'
+    }
+
+    It "Remove-Alias should throw on out-of-range scope"{
+        {
+            Set-Alias -Name "foo" -Value "bar" -ErrorAction Stop
+            Remove-Alias -Name "foo" -Scope 99999 -ErrorAction Stop
+        } | ShouldBeErrorId "ArgumentOutOfRange,Microsoft.PowerShell.Commands.RemoveAliasCommand"
+    }
```

Comment:
Add a case for `ral`

Reply:
@adityapatwardhan We have a test set for aliases in separate file and shouldn't test aliases here.

Reply:
Correct. Please add tests here: https://github.com/PowerShell/PowerShell/tree/master/test/powershell/Modules/Microsoft.PowerShell.Utility

There are test files for Get-Alias and Set-Alias over there.

Reply:
@adityapatwardhan @iSazonov sorry, I saw this conversation just now, I'll move the test with the alias, but I'm not sure I understood where I should put them. Currently I have the test together with the other tests unter https://github.com/PowerShell/PowerShell/tree/master/test/powershell/Modules/Microsoft.PowerShell.Utility in a separate file called "Remove-Alias.Tests.ps1". Should I move the whole file or just the test testing for the ral alias?

Reply:
@PowershellNinja You are right. Changes in DefaultCommands.Tests.ps1 should have you covered to testing alias. Sorry for the confusion.

Reply:
@adityapatwardhan absolutely no problem, I just was not sure if I have to put the test elsewhere. I removed it in the recent commit.